src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -342,9 +350,7 @@ export class OrganizationIntegrations extends AsyncComponent<
   renderSentryApp = (app: SentryApp) => {
     const {organization} = this.props;
     const status = getSentryAppInstallStatus(this.getAppInstall(app));
-    const categories = ['internal', 'unpublished'].includes(app.status)
-      ? [app.status]
-      : getCategories(app.featureData);
+    const categories = getCategoriesForIntegration(app);
```

Comment:
@NisanthanNanthakumar I think it might be better to use `getCategoriesForIntegration` where we use `getCategories` for the other integrations to be more consistent but I'll leave that to you as an optional suggestion